### PR TITLE
Tests for repoquery -f, -l

### PR DESCRIPTION
**Pull request by jan-kolarik**
Related: https://github.com/rpm-software-management/dnf/pull/2083.

### CODE REVIEW

**Comment by jan-kolarik** [comment]
After I've created the tests for dnf4 branch, I realized these are already present in the dnf5, so no need to backport it there.

**Comment by m-blaha** [comment]
Just thinking whether it would have some maintenance benefit to keep `repoquery/files.feature` the same (or as similar as possible) for dnf-4-stack and main branches. What do you think?

File: dnf-behave-tests/dnf/repoquery/files.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Scenario: List of files in the package
+  Given I successfully execute dnf with args "repoquery -l filedep-dependency"
+   Then stdout is
+    """
+    /dummy/file
+    /filedep-dependency-1.0.spec
+    """
+
+
+Scenario: Query packages that owns a file
```
Comment (m-blaha) [inline dnf-behave-tests/dnf/repoquery/files.feature:18]: please add `@bz2276012` tag

**Comment by jan-kolarik** [comment]
> Just thinking whether it would have some maintenance benefit to keep `repoquery/files.feature` the same (or as similar as possible) for dnf-4-stack and main branches. What do you think?

You're right, I'll try to reuse the tests from dnf5, drop the unusable ones and add the missing test for -l in dnf5.

**Comment by m-blaha** [comment]
The `-l` is not completely missing - it just uses long variant of the option `--files`.

**Comment by jan-kolarik** [comment]
I've backported the tests from the dnf5 branch and did small adjustments.

**Comment by jan-kolarik** [comment]
Hmm, it seems the CI tests are failing because F39 environment is used for testing, though the filelists changes were introduced in >= F40.

**Comment by jan-kolarik** [comment]
I think we should better migrate also the dnf4 branch of CI stack to use TMT testing, so we get the different chroots testing "for free".

**Comment by jan-kolarik** [comment]
Hmm, so I forgot to actually add the testing repository that is used in the tests. Anyway, it's interesting no error about that non-existing repo was thrown.

**Review by m-blaha — APPROVED** [review]
Thanks!

**Pull request merged**